include/linux/bpf.h

```diff
@@ -87,6 +90,15 @@ typedef struct
     uint32_t link_fd; ///< File descriptor of link to detach.
 } bpf_link_detach_attr_t;
 
+/// Attributes used by BPF_PROG_ATTACH/DETACH .
+typedef struct
+{
+    uint32_t prog_fd;       ///< File descriptor of program to attach to.
+    uint32_t target_fd; ///< eBPF target to attach/detach to/from.
+    enum bpf_attach_type prog_type; ///< Type of program to attach/detach to/from.
+    uint32_t flags; ///< Flags affecting the attach operation.
```

Comment:
nit: Either make all these comments line up, or make all comments only have 1 space (line 96 doesn't).  I think the former is what the code formatter usually does.